content/docs/tools/piper.md

```diff
@@ -0,0 +1,43 @@
++++
+title='Project "Piper"'
++++
+
+[Project "Piper"][piper] is a set of ready-made Continuous Delivery pipelines for direct use in your project, maintained by [SAP][sap].
```

Comment:
```suggestion
[Project "Piper"][piper] is a set of ready-made Continuous Delivery Jenkins pipelines for direct use in your project, maintained by [SAP][sap] and powered by Cloud Native Buildpacks.
```

Reply:
Well, Project Piper is a whole CI/CD library. While it uses the CNB lifecycle binaries, "Piper... powered by cloud native buildpacks" seems inaccurate.

Do you think the page currently lack attribution to the cloud native buildpacks project and community?

Reply:
Please feel free to rewrite it. I would just prefer it so that the summary displays some connection as to how it's related to the project.

For eg, here's how gitlab and circleci summaries are written -

> CircleCI is a continuous integration and delivery platform. The CNB project maintains an integration, called an orb, which allows users to run pack commands inside their pipelines.

> Gitlab is a web based DevOps platform. It uses pack as part of the Auto DevOps feature, to build applications prior to deploying them.

Reply:
Mmh, then I guess we need to take the initial paragraph of the next header into the summary. Basically, Project Piper now implements the CNB Platform spec as a step in the Jenkins library. What do you think?

Reply:
We have rephrased it, could you have another look?